js/src/components/modals/MultipleKeywords.js

```diff
@@ -0,0 +1,65 @@
+/* global yoastMultipleKeywordsModalL10n */
+import React from "react";
+import styled from "styled-components";
+import interpolateComponents from "interpolate-components";
+
+import IntlProvider from "../IntlProvider";
+import Icon from "yoast-components/composites/Plugin/Shared/components/Icon";
+import YoastSeoIcon from "yoast-components/composites/basic/YoastSeoIcon";
+import { makeOutboundLink } from "yoast-components/utils/makeOutboundLink";
+import PremiumBenefitsList from "../PremiumBenefitsList";
+
+let localizedData = null;
+if ( window.yoastMultipleKeywordsModalL10n ) {
+	localizedData = yoastMultipleKeywordsModalL10n;
+}
+
+const YesYouCanLink = makeOutboundLink();
```

Comment:
`YesYouCanLink` doesn't feel very descriptive (and very Obama campaign slogan-ish).